iquip/apps/scheduler.py

```diff
@@ -0,0 +1,333 @@
+        self.args = [QLabel(f"{key}: {value}", self) for key, value in info.arginfo.items()]
+        self.editButton = QPushButton("EDIT")
+        self.editButton.clicked.connect(self.edit)
+        # layout
+        layout = QHBoxLayout(self)
+        self.argsLayout = QHBoxLayout()
+        self.argsLayout.addWidget(self.nameLabel)
+        for widget in self.args:
+            self.argsLayout.addWidget(widget)
+        layout.addLayout(self.argsLayout, 5)
+        layout.addWidget(self.editButton, 1)
+        self.setLayout(layout)
+
+    def updateInfo(self, info: ExperimentInfo):
+        """Updates the information by modification.
+        
+        The function edit() uses this for updating the edited arguments.
+
+        Args:
+            info: The experiment information.
+        """
+        _dismiss_items(self.argsLayout)
+        self.experimentInfo = info
+        self.nameLabel.setText(info.name)
+        for key, value in info.arginfo.items():
+            self.argsLayout.addWidget(QLabel(f"{key}: {value}", self))
+
+    def data(self) -> ExperimentInfo:
+        """Returns data to ExperimentDelegate for displaying."""
+        return self.experimentInfo
+
+    def edit(self):
+        """Shows frame for edition of experiment information.
+
+        It is called when editButton is pressed.
+        """
+        # TODO(giwon2004): Display a frame for editing the values.
+        print("Button Clicked")
+
+
+class ExperimentModel(QAbstractListModel):
+    """Model for managing the data in the submitted experiment list.
+    
+    Attributes:
+        experimentData: The list of ExperimentView widget.
+    """
+
+    def __init__(self, data: List[ExperimentInfo], parent: Optional[QWidget] = None):
+        """Extended."""
+        super().__init__(parent)
+        self.experimentData = data
+
+    def rowCount(self, parent: Optional[QModelIndex] = QModelIndex()) -> int:
+        """Overridden."""
+        return len(self.experimentData)
+
+    def data(self,
+        index: QModelIndex,
+        role: Optional[Qt.ItemDataRole] = Qt.DisplayRole
+    ) -> ExperimentView:
+        """Overridden."""
+        return self.experimentData[index.row()]
+
+    def supportedDropActions(self) -> int:
+        """Overridden."""
+        return Qt.CopyAction | Qt.MoveAction
+
+    def mimeData(self, index: QModelIndex) -> QMimeData:
+        """Fetches the index of the selected element.
+        
+        Args:
+            index: The ModelIndex instance containing 
+              row, column, and etc. values of the selected element.
+        
+        Returns:
+            A QMimeData instance that holds str value of the index.
+        """
+        mime = super().mimeData(index)
+        mime.setText(str(index[0].row()))
+        return mime
+
+    def dropMimeData(self,  # pylint: disable=too-many-arguments
+        mimedata: QMimeData,
+        action: QAction,
+        row: int,
+        column: int,
+        parentIndex: QModelIndex
```

Comment:
Why are they here? Is this method also overridden?
If so, please modify the docstring and add the pylint disable comments.